app/src/main/java/org/simple/clinic/storage/monitoring/DelegateableSQLiteOpenHelper.kt

```diff
@@ -0,0 +1,42 @@
+package org.simple.clinic.storage.monitoring
+
+import androidx.sqlite.db.SupportSQLiteOpenHelper
+
+class DelegateableSQLiteOpenHelper(
```

Comment:
Should we mark this as a private constructor so that we only use the `Factory` function for creating the helper?

Reply:
Good point, I'll make this change in the next PR.